dataland-frontend/.env

```diff
@@ -1,6 +1,4 @@
-VUE_APP_BASE_API_URL=""
-VUE_APP_API="/api"
-CYPRESS_API="/api"
-CYPRESS_TEASER_COMPANY_PERMIDS=${TEASER_COMPANY_PERMIDS}
+CYPRESS_TEASER_COMPANY_PERM_IDS=${TEASER_COMPANY_PERM_IDS}
```

Comment:
Frage zu anderen Datei. Passt hier aber irgendwie thematisch hin. In dem .env.template steht auch noch TEASER_COMPANY_PERM_IDS. Brauchen wir das da überhaupt noch?

Reply:
habs gelöscht.